### DISCUSSION TITLE
Exception: 无法找到名为“AddGeometry”的资源。资源名称区分大小写。

### DISCUSSION THREAD

**Discussion by 3bDjrvHs50kiZIJb5** [HandyOrg/HandyControl#1268] [Q&A]
![image](https://user-images.githubusercontent.com/45892397/193281022-5f163870-e9c5-4383-8ae6-6ffd2e906f9f.png)

这个是为什么呢?

**Comment by JeremyWu917** [comment]
@zq535228
Hi there,
You can use it like below code if you using `HandyControl`:
```C#
<Button
    Margin="5"
    hc:IconElement.Geometry="{DynamicResource AddGeometry}"
    Content="Add"
    Style="{StaticResource ButtonPrimary}" />
```

Hope it works.
Thanks

**Comment by NaBian** [comment]
请确认引用了资源样式字典。
